<?xml version="1.0"?>
<launch>
 
    <param name="pcTopic"     type="str"    value="/ouster/points" />
    <!-- Los siguientes valores se recomienda NO MODIFICAR hasta hacer pruebas con varios escenarios -->
    <param name="maxlen"      type="double" value="8.0" />  <!-- distancia maxima de la pcl para tomar encuenta-->
    <param name="minlen"      type="double" value="0.5" />  <!-- distancia minima de la pcl para tomar encuenta-->
    <param name="maxZ_filter" type="double" value="0.1" />  <!-- altura maxima de la PCL -->
    <param name="voxel"       type="double" value="0.15" /> <!-- voxelizado de la nube de puntos 0.15 --> 
    <param name="smNorm"      type="double" value="0.4" />  <!-- normales pequeÃ±as  de la pcl 0.4 -->
    <param name="laNorm"      type="double" value="0.56" /> <!-- normales largas  de la pcl 0.56 -->   
    <param name="threshold"   type="double" value="0.02" /> <!-- threshold de filtrado de la ambas normales -->
    <param name="scaleRadio"  type="double" value="0.98" /> <!-- escalado del radio para eliminar ruido en el radio -->

    <param name="vecinos"   type="double" value="10.0" /> <!-- 10 -->
    <param name="th_meanK"  type="double" value="1.0" /> <!-- 1.0 -->



    <node pkg="lidar_obstacle_detection"   type="ouster_obstacle_filtering_node" name="ouster_obstacle_filtering_node" output="screen"/>
 
</launch>
